source/exercises/mailroom-part1.rst

```diff
@@ -35,31 +37,32 @@ Write a small command-line script called ``mailroom.py``. This script should be
 * The script should prompt the user (you) to choose from a menu of 3 actions:
   "Send a Thank You," "Create a Report," or "quit."
 
-Sending a Thank You
+Send a Thank You
 -------------------
+.. I changed heading to match title of function.
```

Comment:
can you please remove this

Reply:
Do you mean to remove the comment?

Reply:
yes that's what I meant. It looks like a temp comment that should be removed.